### PR TITLE
Add Kubernetes support and simplify Docker build

### PR DESCRIPTION
**Pull request by vnugent**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,34 +1,9 @@
 #Dockerfile for rhq-metrics
 
-FROM jboss/base-jdk:7
+FROM vnguyen/wildfly-jdk8
```
Comment (pilhuhn) [inline docker/Dockerfile]: Is there no jboss/* version available?

File: docker/README.adoc [hunk 2]
```diff
@@ -1,5 +1,8 @@
-==== docker build
-`docker build --force-rm=true --no-cache=true -t rhq-metrics:latest .`
```
Comment (pilhuhn) [inline docker/README.adoc]: Should we keep that for people that only run Docker and not Kubernetes?

Comment (vnugent) [inline docker/README.adoc]: Right.  I'll update the Readme with instructions to launch Cassandra container and link with rhq-metrics the plain Docker way.

File: docker/Dockerfile (hunk 1 above)
Comment (vnugent) [inline docker/Dockerfile]: I could be more specific with WildFly version.  vnguyen/wildfly-jdk8 image was built from 8.2.0.Final.

**Pull request merged**